mentorship iosTests/HomeTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  HomeTests.swift
+//  Created on 26/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import XCTest
+@testable import mentorship_ios
+
+class HomeTests: XCTestCase {
+    // custom urlsession for mock network calls
+    var urlSession: URLSession!
+
+    override func setUpWithError() throws {
+        // Set url session for mock networking
+        let configuration = URLSessionConfiguration.ephemeral
+        configuration.protocolClasses = [MockURLProtocol.self]
+        urlSession = URLSession(configuration: configuration)
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
```

Comment:
Ideally, you should deallocate anything you initialise in setUp in the teardown function. 

you could: 
`urlSession = nil`
`configuration = nil`
`super.tearDownWithError()`

Reply:
Okay

Reply:
@ApheleiaS Done. Thanks!